gateway/passthrough-sasl-ssl/gateway.yaml

```
@@ -0,0 +1,39 @@
+apiVersion: platform.confluent.io/v1beta1
+kind: Gateway
+metadata:
+  labels:
+    app.kubernetes.io/name: confluent-for-kubernetes
+    app.kubernetes.io/managed-by: kustomize
+  name: confluent-gateway
+spec:
+  replicas: 3
+  image:
+    application: confluentinc/cpc-gateway:1.0.0-1-ubi9
+    init: confluentinc/confluent-init-container:3.1.0
+  streamingDomains:
+    - name: passthrough-sasl-ssl-kafka
+      type: kafka
+      kafkaCluster:
+        bootstrapServers:
+          - id: passthrough_SASL_SSL
```

nit: 
```suggestion
          - id: SASL_SSL
```
just to show customer the best practices. `listener protocol` basis separation for bootstrapServers.

---

Done.